Ginger/GingerCore/Drivers/JavaDriverLib/JavaDriver.cs

```diff
@@ -518,6 +527,28 @@ private PayLoad HandlePOMElememntExecution(ActUIElement act)
             return response;
         }
 
+        private void IntializeIfWidgetsElement(ElementInfo currentPOMElementInfo)
+        {
+            if (IsPOMWidgetElement(currentPOMElementInfo))
+            {
+                var path = currentPOMElementInfo.Properties.Where(x => x.Name.Equals(ElementProperty.ParentBrowserPath)).FirstOrDefault();
+                if (path != null && !string.IsNullOrEmpty(path.Value))
+                {
+                    InitializeBrowser(new JavaElementInfo() { XPath = path.Value });
+                }
+            }
+        }
+
+        private static bool IsPOMWidgetElement(ElementInfo currentPOMElementInfo)
```

Comment:
I see this method is duplicated on 2 classed. you can put this method on element info or java element info itself call it from there